ci/resources.yml

```diff
@@ -3,16 +3,17 @@ resources:
   type: git
   source:
     branch: master
-    private_key: ((svc-dip.private_key))
-    uri: git@github.ucds.io:dip/aws-management-infrastructure.git
+    uri: https://((dataworks.enterprise_github_url))/dip/aws-management-infrastructure.git
+    username: ((dataworks.enterprise_github_username))
+    password: ((dataworks-secrets.enterprise_github_pat))
   check_every: 720h
```

Comment:
webhooks don't work from GitHub enterprise, so need to reduce the check time, maybe 5 min?

Reply:
Thanks, done